examples/misc/custom_projection.py

```diff
@@ -174,8 +174,8 @@ def _set_lim_and_transforms(self):
 
     def _get_affine_transform(self):
         transform = self._get_core_transform(1)
-        xscale, _ = transform.transform_point((np.pi, 0))
-        _, yscale = transform.transform_point((0, np.pi / 2.0))
+        (xscale, _), (_, yscale) = transform.transform(
```

Comment:
Not sure if this change is worth it. Probably slightly faster but less readable.

Reply:
unchanged